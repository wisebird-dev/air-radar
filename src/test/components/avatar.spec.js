import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Avatar from '../../components/avatar'

test('Avatar renders correctly', async () => {
  const tree = await renderer.create(<Avatar />).toJSON()

  expect(tree).toMatchSnapshot()
})