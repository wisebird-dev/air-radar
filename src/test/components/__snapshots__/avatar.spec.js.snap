// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders correctly 1`] = `
.c0 {
  border-radius: 50%;
  width: 128px;
  height: 128px;
  background-color: #fff;
  padding: 4px;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

<img
  className="c0"
/>
`;
